addons/packages/packetfence.spec

```diff
@@ -302,6 +302,9 @@ Requires: perl(DateTime::TimeZone)
 Requires: samba-winbind-clients, samba-winbind
 Requires: libdrm >= 2.4.74
 Requires: netdata, fping, MySQL-python
+#OpenVAS
+Requires: openvas-cli
+Requires: openvas-libraries
```

Comment:
Its not in our convention to not install the necessary packages for making PF work.

Like we install WMI clients, samba clients, and countless other dependencies so things work as out of the box as possible

Reply:
@julsemaan, I know but packages are available in official repos not in EPEL.

Reply:
samba and krb5-workstation are too and we're not asking people to install them manually so that the AD integration works... Same goes for all the other integrations we do have. Whether its in EPEL, base or zammrepo doesn't make a difference.